### ISSUE TITLE
`<Pointout />` Close icon lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#591]
![d7f4a73c-7dbb-11ec-b537-47d327255353](https://user-images.githubusercontent.com/3180185/159581618-fd6e5ae0-8848-4467-a1ee-e9dacd259965.png)

<dl><dt>Description</dt><dd>Parts of graphics (required to understand the content) do not have a contrast ratio of 3 to 1 against adjacent color(s).
In Dark theme -
Close icons on Pointout page lacks 3:1 color contrast ratio. 
This issue occurs on all close icons present on page.
ForeGround color: #FFFFFF
Background color: #F7846C
CC ratio:  2.5:1</dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
Parts of graphics (required to understand the content) MUST have a contrast ratio of 3 to 1 against adjacent color(s). Exceptions exist.

Parts of graphics (required to understand the content) MUST have a contrast ratio of 3 to 1 against adjacent color(s). Exceptions exist.

HOW TO FIX: 
Fix this issue by adjusting the parts of graphics (that are required to understand the content) and/or adjacent color to increase the contrast to at least 3 to 1.

REFERENCE: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

BACKGROUND: 
People who have low vision or are colorblind may have difficulty perceiving the important parts of an informative graphical object (i.e., the parts of the graphic required to understand the content) if the contrast between parts of the graphic essential for understanding the graphic is insufficient. When an graphical object has adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive the information conveyed by the object.
NOTE: The term "graphical object" applies to stand-alone icons such as a print icon (with no text), and the important parts of a more complex diagram such as each line in a graph. For simple graphics such as single-color icons the entire image is a graphical object. Images made up of multiple lines, colors and shapes will be made of multiple graphical objects, some of which are required for understanding. Not every graphical object needs to contrast with its surroundings - only those that are required for a user to understand what the graphic is conveying. This requirement does not apply to a graphic whose information is available in another form such as text - for example a printer icon with the word "print" next to it - and it does not apply to decorative images that do not convey information.</dd><dt>Source code</dt><dd>

```
<svg xmlns="http://www.w3.org/2000/svg" overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" height="24" width="24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z" vector-effect="non-scaling-stroke" fill="currentColor"></path></svg>
```
</dd><dt>Checkpoint</dt><dd>Non-Text Contrast - Graphical Objects (1.4.11.c)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/Pointout</dd></dl>

**Comment by okry** [comment]
Related to #568
